### ISSUE TITLE
[Feat] connection pool

### ISSUE THREAD

**Issue by Coachonko** [patrickpissurno/vredis#18]
Connection pool is needed.
It could be left to users to implement but every other redis client provides pooling support so vredis probably should too.

**Comment by patrickpissurno** [comment]
Agreed! Similarly to #17, I don't have the time to work on this right now, but feel free to open up a PR and I'll make sure to find some time to review it and get it merged quickly

**Comment by Coachonko** [comment]
> Agreed! Similarly to #17, I don't have the time to work on this right now, but feel free to open up a PR and I'll make sure to find some time to review it and get it merged quickly

I opened these issues as TODO list entries, I think it could help to keep track of what to do. If this is a bad idea feel free to close them.
